internal/services/containerapps/container_app_environment_resource_test.go

```diff
@@ -64,6 +64,21 @@ func TestAccContainerAppEnvironment_complete(t *testing.T) {
 	})
 }
 
+func TestAccContainerAppEnvironment_withWorkloadProfile(t *testing.T) {
```

Comment:
Can we add some update testing?
We'll want coverage on adding and removing a profile to an environment, as well as removing all user specified profiles.

Reply:
Will update the test.